src/meta/processors/partsMan/DropSpaceProcessor.cpp

```diff
@@ -41,6 +41,16 @@ void DropSpaceProcessor::process(const cpp2::DropSpaceReq& req) {
     deleteKeys.emplace_back(MetaServiceUtils::indexSpaceKey(req.get_space_name()));
     deleteKeys.emplace_back(MetaServiceUtils::spaceKey(spaceId));
 
+    // delete related role data.
```

Comment:
Now we just delete the parts information when deleting space.  We could delete the useless user and schema when compaction.

Reply:
Yeah, that makes sense.  Let me modify it later.